common/spec/dependabot/clients/codecommit_spec.rb

```diff
@@ -4,17 +4,11 @@
 require "spec_helper"
 require "dependabot/clients/codecommit"
 
+ENV["AWS_REGION"] = "us-east-1"
```

Comment:
Missed this when testing locally b/c I had already had it set.. If the preference is to have env vars stubbed per test or established using a different method please let me know.